FoodFite/Models/UserProfile.cs

```diff
@@ -13,11 +13,21 @@ namespace FoodFite.Models
     public class UserProfile
     {
         public string Name { get; set; }
+        public string Opponent { get; set; }
+        public string Weapon { get; set; }
         public List<Item> Inventory { get; set;}
         public double Health { get; set;}
         public Protection Clothes { get; set; }
+        public Dictionary<string, Food> FoodMap { get; set; }
+        public List<Food> FoodInventory{get; set;}
+        public Protection Protection{get; set;}
```

Comment:
This looks to be the same as clothes

Reply:
yeah this was due to us both working the model at the same time. I'll update this to use your fields.